web/static/js/components/surveys/SurveyWizardScheduleStep.jsx

```diff
@@ -125,7 +125,9 @@ class SurveyWizardScheduleStep extends Component {
           : ''
         }
         <div className='row'>
-          <TimezoneDropdown selectedTz={survey.schedule.timezone} onChange={this.updateTimezone} readOnly={readOnly} />
+          <div className='col s12 m6'>
+            <TimezoneAutocomplete selectedTz={survey.schedule.timezone} />
```

Comment:
```suggestion
            <TimezoneAutocomplete selectedTz={survey.schedule.timezone} readOnly={readOnly} />
```
`TimezoneAutocomplete` seems to be missing a `readOnly` prop to only display the data without allowing editions. That's needed for the `Settings` tabs in a running survey.